README.md

```diff
@@ -3,6 +3,17 @@ messybrainz-server
 
 The server components for the MessyBrainz project.
 
+MessyBrainz is a [MetaBrainz](https://metabrainz.org) project to support unclean metadata. While
+[MusicBrainz](https://musicbrainz.org) is designed to link clean metadata to stable identifiers,
+there is a need to identify unclean or misspelled data as well. MessyBrainz provides identifiers to
+unclean metadata, and where possible, links it to stable MusicBrainz identifiers.
+
+MessyBrainz is currently used in support of two projects, [ListenBrainz](https://listenbrainz.org)
+and [AcousticBrainz](https://acousticbrainz.org). Submission to MessyBrainz is restricted, however
+the resulting data will be made freely available.
+
+[Website](https://messybrainz.org) | [Beta Website](https://beta.messybrainz.org)
```

Comment:
As you can't actually download anything from the website, does it make sense to show the beta link? I would just have the link to the main site.